network-area-diagram-viewer/src/main/resources/svg.js

```diff
@@ -0,0 +1,448 @@
+
+// TODO(Luma) consider three winding transformers
+// TODO(Luma) consider nodes with multiple buses
+// TODO(Luma) separate components of edges, having one that can be "scaled" (length line from exit of node to middle point)
+//              for reference, see Android NinePatch drawables
+//              https://developer.android.com/develop/ui/views/graphics/drawables#nine-patch
+// TODO(Luma) JavaFX seems to have problems drawing a translated foreign object, try to enclose it in a <g> element?
```

Comment:
I don't see any problems, what is it? If the problem occurs only in javaFX, we should not really care about it.

Reply:
Saw it in macOS, when I move the nodes with bus legend visible there is a trace left:
<img width="322" alt="Screenshot 2022-11-29 at 10 04 42" src="https://user-images.githubusercontent.com/3374819/204486836-246bde3d-7309-479a-b348-0ade8d597d17.png">

Reply:
ah, yes that's sometimes happening on my side also, but only on javafx webview indeed. Let's forget this, javafx webview won't be used for this feature except here.